doc/guides/method.rdoc

```diff
@@ -0,0 +1,35 @@
+= Documenting a Method
+
+== Structure
+
+=== Synopsis
+
+Begin with a one-line synopsis that tells what the method does.  Example:
+  *  call-seq:
+  *    array <=> other_array -> -1, 0, or 1
+  *
+  *  Returns -1, 0, or 1 as +self+ is less than, equal to, or greater than +other_array+.
+
+== Multiple Signatures
+
+\[Is 'signatures' the correct term?]
+
+When a method has multiple signatures,
+document each signature separately, as if it were a distinct method.
```

Comment:
I'm not sure this is a good general rule, though I think it does make sense in this case.  Whether the signatures should be documented separately depends on how much overlap there is between them.